mne/epochs.py

```diff
@@ -474,8 +474,14 @@ def ch_names(self):
         return self.info['ch_names']
 
     def plot(self, epoch_idx=None, picks=None, scalings=None,
-             title_str='#%003i', show=True, block=False):
-        """Visualize single trials using Trellis plot.
+             title_str='#%003i', show=True, block=False, n_epochs=20,
+             n_channels=10, fig_title=None, trellis=True):
+        """Visualize single trials.
+
+        Bad epochs can be marked with a left click on top of the epoch.
```

Comment:
This is only for `trellis=False`. You should also document how to use `trellis=True`.